templates/endpoints_v2/README.ipynb

```diff
@@ -20,7 +20,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "From the terminal (press Ctrl + ` to open terminal)  use the Ray Serve CLI to deploy a model. It will be run locally in this workspace's cluster:\n",
+    "From the terminal (press Ctrl + ` to open terminal in VSCode), use the Ray Serve CLI to deploy a model. It will be run locally in this workspace's cluster:\n",
```

Comment:
I remembered that Japneet was a bit confused about whether he should open vs code terminal or laptop terminal or semething. Not sure if we need to do anything here.

Reply:
Yeah good point, reworded it to avoid this confusion.